org.eclipse.winery.common/pom.xml

```diff
@@ -104,6 +124,12 @@
             <version>${ch.qos.logback.logback-classic.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>RELEASE</version>
```

Comment:
This version should be the same as elsewhere: Please use `${junit.jupiter.version}`.